pilot/pkg/kube/inject/webhook.go

```diff
@@ -188,12 +188,18 @@ func NewWebhook(p WebhookParameters) (*Webhook, error) {
 // Run implements the webhook server
 func (wh *Webhook) Run(stop <-chan struct{}) {
 	go func() {
-		if err := wh.server.ListenAndServeTLS("", ""); err != nil {
-			log.Errorf("ListenAndServeTLS for admission webhook returned error: %v", err)
+		err := wh.server.ListenAndServeTLS("", "")
+
+		msg := fmt.Sprintf("Stopped listening on %s", wh.server.Addr)
+		select {
+		case <-stop:
+			log.Infof(msg)
+		default:
+			log.Fatalf("%s due to error: %v", msg, err)
```

Comment:
Can we copy the approach taken by the validation webhook? i.e.
https://github.com/istio/istio/blob/efd8e8478248dca2cf0c15c908583552adc235be/galley/pkg/crd/validation/webhook.go#L211-L215

Reply:
Yes, we can. Will do that.